clientSrc/components/Login.js

```diff
@@ -1,32 +1,32 @@
-import React, {useState} from 'react'
-import '../styles/Login.less'
+import React, { useState } from 'react';
+import '../styles/Login.less';
 
 const Login = () => {
     const [username, setUsername] = useState('');
     const [password, setPassword] = useState('');
 
     const handleUsernameChange = (e) => {
-        setUsername(e.target.value)
-    }
+        setUsername(e.target.value);
+    };
 
     const handlePasswordChange = (e) => {
-        setPassword(e.target.value)
-    }
+        setPassword(e.target.value);
+    };
 
     const loginUser = (e) => {
-        //This doesn't do anything yet. Will check username and password 
-        //against the database, and allow access if match.
-        e.preventDefault()
-        console.log(username)
-        console.log(password)
-    }
+        // This doesn't do anything yet. Will check username and password
```

Comment:
no need to uncomment these since it was included in the previous merge and you didn't note that you are removing it. It's little bit confusing since I was looking in console and didn't see the output I was expecting

Reply:
Uncommented. The reason I'd commented them out is because ES lint raises and error when they're in. I looked at the documentation but it wasn't clear how to fix this error. Do you now how to stop the ES Lint error with console.log statements?

![Screen Shot 2020-12-29 at 2 01 26 PM](https://user-images.githubusercontent.com/68879246/103316805-78039480-49de-11eb-9321-b7fd270b9354.png)

Reply:
hm, not really, Ive never worked with eslint like  this